cmd/buildah/bud.go

```diff
@@ -203,6 +203,9 @@ func budCmd(c *cli.Context) error {
 		Runtime:             runtime,
 		RuntimeArgs:         runtimeFlags,
 	}
+	if !quiet {
+		options.ReportWriter = os.Stderr
```

Comment:
Right, but in practice a script will probably still want to use --quiet to avoid the stderr spew.